src/templates/eolAnnouncement.js

```diff
@@ -0,0 +1,228 @@
+      <SEO
+        location={location}
+        title={title}
+        type={TYPES.EOL}
+        disableSwiftype={disableSwiftype}
+      />
+      <PageTitle
+        css={css`
+          max-width: 850px;
+          margin-bottom: 0.5rem;
+          line-height: 1.15;
+        `}
+      >
+        {title}
+      </PageTitle>
+      <div
+        css={css`
+          max-width: 850px;
+        `}
+      >
+        <DateIcon icon="fe-calendar" dateString={publishDate}>
+          Published:
+        </DateIcon>
+        <DateIcon icon="fe-minus-circle" dateString={eolEffectiveDate}>
+          EOL effective:
+        </DateIcon>
+
+        <div
+          css={css`
+            color: var(--secondary-text-color);
+            font-size: 1.125rem;
+            line-height: 1.75;
+            padding-bottom: 1rem;
+            border-bottom: 1px solid var(--divider-color);
+            margin: 1rem 0;
+          `}
+        >
+          <p>{summary}</p>
+          {learnMoreLink && (
+            <ul
+              css={css`
+                display: flex;
+                list-style: none;
+                margin: 0;
+                padding: 0;
+                font-size: 0.875rem;
+                gap: 1rem;
+
+                > li {
+                  margin: 0;
+                }
+
+                @supports not (gap: 1rem) {
+                  > li:not(:last-child) {
+                    margin-right: 1rem;
+                  }
+                }
+              `}
+            >
+              <li>
+                <MetaLink to={learnMoreLink} siteUrl={siteUrl}>
+                  Learn more
+                </MetaLink>
+              </li>
+            </ul>
+          )}
+        </div>
+      </div>
+      <Layout.Content
+        css={css`
+          max-width: 850px;
+        `}
+      >
+        <MarkdownContainer dangerouslySetInnerHTML={{ __html: html }} />
+        <ContributingGuidelines
+          css={css`
+            margin-top: 1rem;
+            padding-left: 0;
+            > * {
+              justify-content: flex-start;
+              text-align: left;
+            }
+          `}
+          fileRelativePath={fileRelativePath}
+          issueLabels={['feedback', 'feedback-issue']}
+        />
+      </Layout.Content>
+    </ErrorBoundary>
+  );
+};
+
+EolTemplate.propTypes = {
+  data: PropTypes.object.isRequired,
+  location: PropTypes.object.isRequired,
+  pageContext: PropTypes.object.isRequired,
+};
+
+export const pageQuery = graphql`
+  query($slug: String!) {
+    site {
+      siteMetadata {
+        siteUrl
+      }
+    }
+    markdownRemark(fields: { slug: { eq: $slug } }) {
+      html
+      frontmatter {
+        title
+        eolEffectiveDate(formatString: "MMMM D, YYYY")
+        publishDate(formatString: "MMMM D, YYYY")
+        summary
+        learnMoreLink
+      }
+      fields {
+        fileRelativePath
+      }
+    }
+  }
+`;
+
+const DateIcon = ({ children, icon, dateString }) => {
```

Comment:
Can we make the `DateIcon` and `MetaLink` styled components vs new components? That would get rid of the need for propTypes for them and simplify the file